packages/storage/amplify_storage_s3/example/integration_test/utils/tear_down.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+final _logger = AmplifyLogger().createChild('StorageTests');
+
+/// Adds a tear down to remove the object at [path].
+void addTearDownPath(StoragePath path) {
+  addTearDown(
+    () {
+      try {
+        return Amplify.Storage.remove(path: path).result;
+      } on Exception catch (e) {
+        _logger.warn('Failed to remove file after test', e);
```

Comment:
Q: wonder if we should fail the test when we can't tear down the resource. If there's a failure, we need to investigate the cause, rather than let it happen silently.

Reply:
I added `rethrow` to these. During development it can be nice to catch and log to focus on the test rather than the teardown, but I agree that these failing could indicate an issue.